### PR TITLE
[RPC Gateway] Able to disable DB sync on specific chains

### PR DESCRIPTION
**Pull request by rollingtumbling**
We are adding on/off switch of DB sync for each chain. The config lives in `rpcProviderProdConfig.json` by name `enableDbSync`. It's default value is `false`.

This PR won't make actual prod changes. All DB sync for already launched chains stay the same.

Testing is done in local env. I set some of the chains to `enableDbSync: false`, and some of them `enableDbSync: true`. I can see that although they all serve traffic well using RPC gateway, some chain have DB sync and some chain has none.

Chains with DB sync turned on:

Chains with DB sync turned off:

### CODE REVIEW

**Comment by rollingtumbling** [comment]
* **#565** <a href="https://app.graphite.dev/github/pr/Uniswap/routing-api/565?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rollingtumbling and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/config/rpcProviderProdConfig.json [hunk 1]
```diff
@@ -3,30 +3,35 @@
     "chainId": 42220,
     "useMultiProviderProb": 1,
     "providerInitialWeights": [1, 0],
-    "providerUrls": ["QUICKNODE_42220", "INFURA_42220"]
+    "providerUrls": ["QUICKNODE_42220", "INFURA_42220"],
+    "enableDbSync": true
```
Comment (jsy1218) [inline lib/config/rpcProviderProdConfig.json:7]: Shouldn't `enableDbSync` be false..?

Comment (rollingtumbling) [reply lib/config/rpcProviderProdConfig.json:7]: Sorry this PR is to only support disabling DB sync. We have other PRs to actually disable DB sync for certain chains.

Comment (rollingtumbling) [reply lib/config/rpcProviderProdConfig.json:7]: PR title is updated.

**Review by jsy1218 — APPROVED** [review]

**Pull request merged**